packages/tabs/src/TabPanel/TabPanel.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useMemo } from 'react';
+
+import {
+  useDescendant,
+  useDescendantsContext,
+} from '@leafygreen-ui/descendants';
+
+import {
+  TabPanelsDescendantsContext,
+  TabsDescendantsContext,
+} from '../Tabs/Tabs';
```

Comment:
nit: it might make more sense to either lift these up one directory level, or move them into e.g. `/TabContext`